src/parser/RdfEscaping.cpp

```diff
@@ -162,29 +162,36 @@ std::string escapeNewlinesAndBackslashes(std::string_view literal) {
   return absl::StrReplaceAll(literal, {{"\n", "\\n"}, {R"(\)", R"(\\)"}});
 }
 
+// ____________________________________________________________________________
+void literalUnescape(std::string_view input, std::string& res) {
```

Comment:
functions that only appear internally in the `.cpp` file and are not public in the header should be marked `static`

Reply:
(there are some of those)>